src/templates/finops-hub/modules/dataFactory.bicep

```
@@ -4985,10 +4985,148 @@ resource pipeline_ExecuteQueries 'Microsoft.DataFactory/factories/pipelines@2018
+        }
+        userProperties: [ ]
+        typeProperties: {
+          variableName: 'UniqueManifestPaths'
+          value: {
+            value: '@union(variables(\'ManifestPaths\'), variables(\'ManifestPaths\'))'
+            type: 'Expression'
+          }
+        }
+      }
+      { // Generate Manifest Blobs
+        name: 'Generate Manifest Blobs'
+        type: 'ForEach'
+        dependsOn: [
+          {
+            activity: 'Distinct Manifest Data'
+            dependencyConditions: [
+              'Succeeded'
+            ]
+          }
+        ]
+        userProperties: []
+        typeProperties: {
+          items: {
+            value: '@variables(\'UniqueManifestPaths\')'
+            type: 'Expression'
+          }
+          batchCount: 2
+          isSequential: false
+          activities: [
+            { // Create Manifest
+              name: 'Create Manifest'
+              description: 'Create a manifest file in the ingestion container to trigger ADX ingestion'
+              type: 'Copy'
+              dependsOn: []
+              policy: {
+                timeout: '0.12:00:00'
+                retry: 0
+                retryIntervalInSeconds: 30
+                secureInput: false
+                secureOutput: false
+              }
+              userProperties: []
+              typeProperties: {
+                source: {
+                  type: 'JsonSource'
+                  storeSettings: {
+                    type: 'AzureBlobFSReadSettings'
+                    recursive: true
+                    enablePartitionDiscovery: false
+                  }
+                  formatSettings: {
+                    type: 'JsonReadSettings'
+                  }
+                }
+                sink: {
+                  type: 'JsonSink'
+                  storeSettings: {
+                    type: 'AzureBlobFSWriteSettings'
+                  }
+                  formatSettings: {
+                    type: 'JsonWriteSettings'
+                  }
+                }
+                enableStaging: false
+              }
+              inputs: [
+                {
+                  referenceName: dataset_config.name
+                  type: 'DatasetReference'
+                  parameters: {
+                    fileName: 'settings.json'
+                    folderPath: {
+                      value: configContainerName
+                      type: 'Expression'
+                    }
+                  }
+                }
+              ]
+              outputs: [
+                {
+                  referenceName: dataset_manifest.name
+                  type: 'DatasetReference'
+                  parameters: {
+                    fileName: 'manifest.json'
+                    folderPath: {
+                      value: '@concat(\'${ingestionContainerName}/\', split(item(),\'/\')[0], \'/\', utcNow(\'yyyy\'), \'/\', utcNow(\'MM\'), \'/\', utcNow(\'dd\'), \'/\', split(item(),\'/\')[1])'
```

nit: Shouldn't we take the path as an input? I'm assuming the folder path won't always be the same. For instance, reservation recommendations now have an extra folder. I'm guessing these will need a separate folder as well to handle how ADX ingestion manages extents.

Also, I believe you can simplify the path a bit with one utcNow() call:
```suggestion
                      value: '@concat(\'${ingestionContainerName}/\', split(item(),\'/\')[0], \'/\', utcNow(\'yyyy/MM/dd\'), \'/\', split(item(),\'/\')[1])'
```

---

The path follows the same pattern as the queries Parquet outputs. Therefore, it lands on the same folder. 

I simplified the uctNow() call. Thanks!